R/utils.R

```diff
@@ -28,7 +35,7 @@ is_numeric <- function(chars) {
 #'
 #' @examples
 is_rec_from_range <- function(var_details_row) {
-  return (grepl(":", var_details_row$recFrom, fixed=TRUE))
+  return (grepl(",", var_details_row$recFrom, fixed = TRUE))
```

Comment:
I think it would be better to use the `get_margins` method for this. If the margins returned are the same then this method would return TRUE otherwise it would return FALSE.
The `get_margins` method would have to be updated to handle recFrom values that are not intervals.